Marlin/ultralcd_impl_DOGM.h

```diff
@@ -602,7 +625,7 @@ static void lcd_implementation_status_screen() {
   if (PAGE_CONTAINS(STATUS_BASELINE + 1 - INFO_FONT_HEIGHT, STATUS_BASELINE)) {
     u8g.setPrintPos(0, STATUS_BASELINE);
 
-    #if DISABLED(FILAMENT_LCD_DISPLAY)
+    #if DISABLED(FILAMENT_LCD_DISPLAY) || (DISABLED(SDSUPPORT) && ENABLED(FILAMENT_LCD_DISPLAY))
```

Comment:
Well, It tried to keep "original flow", as this happens build time, so no performance issues.